pkg/interface/src/views/components/leap/Omnibox.tsx

```diff
@@ -60,9 +61,10 @@ export function Omnibox(props: OmniboxProps): ReactElement {
 
   const contacts = useMemo(() => {
     const maybeShip = `~${deSig(query)}`;
+    const selflessContactState = omit(contactState, `~${window.ship}`);
     return ob.isValidPatp(maybeShip)
-      ? { ...contactState, [maybeShip]: {} }
-      : contactState;
+      ? { ...selflessContactState, [maybeShip]: {} }
```

Comment:
This pushes the maybeship into the contact state if it's a valid patp, so we'll still show up. 

You want it to be like "if this is a valid patp *and* it's not us, push it in", not removing us as a contact in the state, because we push any valid patp into the list, not just our contacts as an index.